RELEASES.md

```diff
@@ -14,8 +16,7 @@ Version 0.9.2 (2023-05-XX)
 * Ford Explorer Hybrid 2020-22 support
 * Ford Kuga 2020-22 support
 * Ford Kuga Hybrid 2020-22 support
-* Ford Kuga Plug-in Hybrid 2020-22 support
-* Honda HR-V 2023 support thanks to AlexandreSato and galegozi!
+* Ford Kuga Plug-in Hybrid 2020-22 suppor* Honda HR-V 2023 support thanks to AlexandreSato and galegozi!
```

Comment:
I'll take a look and see what I messed up. Sorry about that.

Reply:
Fixed, good catch. No idea how I pulled that off. Pure unadulterated not paying attention to what I was doing.

I do not believe the new route running OP is working properly. I did just grab a new one I can upload if needed.

Reply:
I changed it to use the first segment, since the last one usually is invalid due to the ignition falling -- the car stops sending messages and OP stays on for a few seconds recording that as part of the drive.

Reply:
Sure, if it's a bit longer that sounds good

Reply:
Yeah a longer route would be ideal

Reply:
I'll get one today

Reply:
Still failing! Longer than 10 min? Is there anything specific I should make sure to do during the drive?

Reply:
looks good! I just needed to upload it to our CI bucket